CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ if(CODE_COVERAGE)
 		COMMAND ${LCOV_PATH} --gcov-tool ${GCOV_PATH} -c -i -d . -b ${PROJECT_SOURCE_DIR} -o ${PROJECT_NAME}.base
 	
 	        # Run executables
-		COMMAND ${CMAKE_SOURCE_DIR}/build/bin/Covid19EERAModel
+		COMMAND ${CMAKE_SOURCE_DIR}/build/bin/Covid19EERAModel ${COV_TEST_FLAGS} -s original  -m prediction
```

Comment:
Why do we run coverage in prediction mode? Inference would make more sense since it will exercise almost all of the prediction code plus the inference code (since the prediction framework has very little in it other than the model code).